src-js/app.js

```diff
@@ -1,6 +1,7 @@
 import * as Cookie from 'js-cookie';
 import * as search from './instantsearch.js';
 import * as lang from './lang.js'
+import * as HighlightJSBadge from '../node_modules/highlightjs-badge/highlightjs-badge.min.js'
```

Comment:
Is this the only we can reference that package? Not a big fan of having the relative path to the package directly here

Reply:
Also, side effect: `HighlightJSBadge` is going to be marked as `unused` because it is imported here but not really used. Now I understand why you did that since you need it to be part of the `app.bundle.js` but we should at the very least add a comment of what we are doing here.

Reply:
import * as highlightJSBadge from 'highlightjs-badge';
works